server.js

```diff
@@ -0,0 +1,221 @@
+<link rel="stylesheet" href="/styles.css">
+</head>
+<body>
+    <h1>${parsedInfo.elementName}</h1>
+    <h2>${parsedInfo.elementSymbol}</h2>
+    <h3>${parsedInfo.elementAtomicNumber}</h3>
+    <p>${parsedInfo.elementDescription}</p>
+    <p><a href="/">back</a></p>
+</body>
+</html>`;
+                fs.writeFile(`./public/${parsedInfo.elementName}.html`, template, (err) => {
+                    if (err) {
+                        return console.log(err);
+                    }
+                    console.log('Filed Saved!');
+                })
+                res.writeHead(200, {
+                    'Content-Type': 'application/json'
+                });
+                let newFiles = []
+                fs.readdir('./public/', (err, files) => {
+                    for (let i = 0; i < files.length; i++) {
+                        if (files[i] !== '.keep' && files[i] !== '404.html' && files[i] !== 'css' && files[i] !== 'index.html' && files[i] !== 'styles.css') {
+                            newFiles.push(files[i]);
+                        };
+                    };
+                    let newIndex = `<!DOCTYPE html>
+                    <html lang="en">
+                    
+                    <head>
+                        <meta charset="UTF-8">
+                        <title>The Elements</title>
+                        <link rel="stylesheet" href="/styles.css">
+                    </head>
+                    
+                    <body>
+                        <h1>The Elements</h1>
+                        <h2>These are all the known elements.</h2>
+                        <h3>These are ${newFiles.length} of them.</h3>
+                        <ol>`;
+                    for (let i = 0; i < newFiles.length; i++) {
+                        let newList = `<li>
+                        <a href="/${newFiles[i]}">${newFiles[i].split('.')[0]}</a>
+                    </li>`;
+                        newIndex += newList;
+                    };
+                    let htmlEnd = `    </ol>
+                    </body>
+                    
+                    </html>`;
+                    fs.writeFile('./public/index.html', newIndex + htmlEnd, (err) => {
+                        if (err) {
+                            throw err;
+                        }
+                        console.log('Index.html updated!');
+                    });
+                });
+                res.end();
+            };
+        }
+        // DELETE METHOD
+        if (req.method === 'DELETE') {
+            fs.readdir(`./public/`, (err, files) => {
+                if (files.includes(URL.slice(1)) === false) {
+                    res.writeHead(500, {
+                        'Date': `${date}`,
+                        'Content-Type': `application/json`,
+                    });
+                    res.write(`{ "error" : "resource ${URL} does not exist" }`)
+                    res.end();
```

Comment:
Not putting a return here could end up with a scenario where you go through the if, then go through the rest of the code below as well, causing a `res cannot write head twice` error.